srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java

```diff
@@ -301,6 +304,13 @@ public void restoreCoversUpId(CqnStructuredTypeRef ref, BooksCovers cover) {
 		cover.setUpId((String) analyzer.analyze(ref).rootKeys().get(Books.ID));
 	}
 
+	@On(event = CqnService.EVENT_READ, entity = Info_.CDS_NAME)
```

Comment:
Could be shortened to:
```suggestion
	@On(entity = Info_.CDS_NAME)
```

Reply:
Yeah, or we remove `CdsReadEventContext` from the signature, as it is not used.

Reply:
yes, also fine with me. I didn't recognize the auto merge, sorry.